.prettierignore

```diff
@@ -0,0 +1,2 @@
+dist
+shims
```

Comment:
Why not `shims`?

It seems to be actual source-code (which we eventually bundle into our eventual bundle (right @jawj?)